components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -842,9 +842,23 @@
 }
 .cancer_status_select {
   display: inline-block;
-  width: 100px;
+  width: 90px;
   margin-right: 15px;
 }
+.cancer_age_select {
+  display: inline-block;
+  margin-right: 6px
+}
+.cancer_field .clickable:hover {
+  cursor: pointer;
+}
+.cancer_field label.clickable.disabled {
+  color: #CCC;
+  cursor: initial;
```

Comment:
`initial` is [not supported by IE](http://caniuse.com/#feat=css-initial-value), use one of `auto` or `default`.